rex/external/standard_attrs.json

```diff
@@ -0,0 +1,35 @@
+{
+    "time_index": {
+        "standard_name": "time",
+        "long_name": "time",
+        "calendar": "proleptic_gregorian"
+    },
+    "latitude": {
+        "standard_name": "latitude",
+        "units": "degree_north"
+    },
+    "longitude": {
+        "standard_name": "longitude",
+        "units": "degree_east"
+    },
+    "temperature": {
+        "standard_name": "air_temperature",
+        "units": "C"
+    },
+    "windspeed": {
+        "standard_name": "wind_speed",
+        "units": "m s-1"
+    },
+    "winddirection": {
+        "standard_name": "wind_to_direction",
+        "units": "degree"
+    },
+    "pressure": {
+        "standard_name": "air_pressure",
+        "units": "Pa"
+    },
+    "relativehumidity": {
+        "standard_name": "relative_humidity",
+        "units": 1
```

Comment:
This might create some confusion since the original unit was `percent`, but that is the way how CF defines unitless values, thus how everyone (IPCC, NASA, NOAA, USGS, ...) do that. I think this is the way to move forward. It might help adding a new attribute `comments` describing it in case some one get confused with `mixing_ratio` or any one of the other related quantities.

The advantage of using `standard_name` is that it is a controlled vocabulary, so it has a clear meaning, but not everyone knows that. Anyways, I think it is the way to go.

Reply:
So will this assign units of "1" to our current data that is from 0-100? Is CF standard fractional and ours is percentage?

Reply:
Looks like if it's a fractional quantity then it should be `units=1` ([source](https://cfconventions.org/cf-conventions/cf-conventions.html#dimensionless-units:~:text=A%20variable%20with%20no%20units%20attribute%20is%20assumed%20to%20be%20dimensionless.%20However%2C%20a%20units%20attribute%20specifying%20a%20dimensionless%20unit%20may%20optionally%20be%20included.%20The%20canonical%20unit%20(see%20also%20Section%203.3%2C%20%22Standard%20Name%22)%20for%20dimensionless%20quantities%20that%20represent%20fractions%2C%20or%20parts%20of%20a%20whole%2C%20is%201.)). I think if it's a percent value (i.e. in the range 0-100), then it should be `units="percent"` (or `units="%"`, if that's preferred) according to [this](https://ncics.org/portfolio/other-resources/udunits2/?#percent)

Reply:
So our data is defined as % and when unscaled it's 0-100. Are we changing how we scale the data and presenting units as fractional, or are we still serving 0-100 rh and erroneously saying the units are fractional? Seems like we're caught in between several inconsistencies here, possibly including my simple confusion. 

An aside - why don't we just take the units from the h5 file attributes? all of our H5 files (except maybe some of the test files?) should have a "units" attr attached to the dataset in h5.

Reply:
I'll look into the unit inconsistency for `rh`. Any other datasets you can think of that would have this issue?

I think my plan (assuming it's possible to do) was to grab the attribute from the h5 file and use the units from that, falling back to these hardcoded values if it's missing. Still need to get around to that

Reply:
Circling back to the units discussion - I believe the correct unit here should be `"percent"` since the unscaled values fall between 0-100. Sorry about the confusion here. I will update the config file accordingly